internal/pkg/instrumentation/bpf/google.golang.org/grpc/client/bpf/probe.bpf.c

```diff
@@ -105,7 +108,27 @@ int uprobe_ClientConn_Invoke(struct pt_regs *ctx)
     return 0;
 }
 
-UPROBE_RETURN(ClientConn_Invoke, struct grpc_request_t, grpc_events, events, 3, 0, true)
+SEC("uprobe/ClientConn_Invoke")
+int uprobe_ClientConn_Invoke_Returns(struct pt_regs *ctx) {
+    struct go_iface go_context = {0};
+    get_Go_context(ctx, 3, 0, true, &go_context);
+    void *key = get_consistent_key(ctx, go_context.data);
+    struct grpc_request_t *grpc_span = bpf_map_lookup_elem(&grpc_events, &key);
+    if (grpc_span == NULL) {
+        bpf_printk("event is NULL in ret probe");
+        return 0;
+    }
+    if (is_register_abi()) {
```

Comment:
Do we plan to handle the alternate code path at some point? Or are we just not planning to support the case for pre-Go 1.17?

Reply:
removed this